src/middlewares/ReadingStat.js

```diff
@@ -0,0 +1,36 @@
+import models from '../db/models';
+import Response from '../helpers/response';
+
+const { Art } = models;
+
+/** Reading Stat Class */
+class ReadingStatMiddleware {
```

Comment:
Please ensure to make the file name and export name and class name same.

Reply:
okay, thanks.